src/components/PrivateRoute/index.tsx

```diff
@@ -19,11 +20,19 @@ export function withPublic(Component: NextPageWithLayout) {
 export const withProtected = (Component: NextPageWithLayout) => {
   const PrivateComponent = (props: object) => {
     const { auth } = useAuth()
+    const { data: userData } = useUser()
     const user = auth.currentUser
-    const router = useRouter()
     if (user === null) {
-      router.replace('/login')
+      Router.replace('/login')
     }
+    useEffect(() => {
+      if (
+        userData &&
+        !(userData.info.email && userData.info.phone && userData.info.name)
+      ) {
+        Router.replace('/signupDetails')
```

Comment:
Can we change this route to something more in line with standard convention? e.g. `/profile/sign-up` or similar